keytransform/interface.go

```diff
@@ -16,12 +16,14 @@ type KeyTransform interface {
 type Datastore interface {
 	ds.Shim
 	KeyTransform
+
+	Batch() (ds.Batch, error)
```

Comment:
oh, this change is probably not necessary now that i've changed the constructor to return `*ktds`